modules/core/src/lib/loader-utils/option-utils.ts

```diff
@@ -26,8 +26,7 @@ export function getGlobalLoaderState(): GlobalLoaderState {
   const {loaders} = globalThis;
 
   // Add _state object to keep separate from modules added to globalThis.loaders
-  loaders._state = loaders._state || {};
-  return loaders._state;
+  return loaders._state || (loaders._state = {});
```

Comment:
Nit: I would prefer an if statement rather than going down the dark path of statements used as expressions.
```suggestion
  if (!loaders._state) {
     loaders._state = {};
  }
  return loaders._state;
```